apps/backend/src/db/migrations/20240823125503_add_onboarding_answers_data.ts

```diff
@@ -0,0 +1,77 @@
+import { type Knex } from "knex";
+
+const TABLE_NAME = {
+	ONBOARDING_ANSWERS: "onboarding_answers",
+	ONBOARDING_QUESTIONS: "onboarding_questions",
+} as const;
+
+const ColumnName = {
+	ANSWER: "answer",
+} as const;
+
+const ANSWERS = [
+	[
+		"Family and relationships",
+		"Career and professional growth",
+		"Personal development and learning",
+		"Health and well-being",
+		"Financial stability",
+	],
+	[
+		"Parent or caregiver",
+		"Professional or student",
+		"Partner or spouse",
+		"Friend or community member",
+		"Individual focused on personal growth",
+	],
+	["Yes", "No"],
+	[
+		"Balancing work and personal life",
+		"Maintaining healthy relationships",
+		"Managing stress and well-being",
+		"Pursuing personal goals or hobbies",
+		"Financial planning and stability",
+	],
+	[
+		"Achieve a healthier lifestyle",
+		"Develop stronger connections with loved ones",
+		"Gain clarity in career direction",
+		"Improve financial planning",
+		"Enhance personal or spiritual growth",
+	],
+	[
+		"Reflective journaling",
+		"Physical exercise or wellness routines",
+		"Creative activities (e.g., art, music)",
+		"Setting and achieving small goals",
+	],
+];
+
+async function up(knex: Knex): Promise<void> {
+	await knex.transaction(async (transaction) => {
+		const questions: { id: number }[] = await transaction(
+			TABLE_NAME.ONBOARDING_QUESTIONS,
+		).select("id");
+
+		const answerInserts = questions.flatMap((question, index) => {
+			return ANSWERS[index]?.map((answer) => ({
+				answer,
+				question_id: question.id,
+			}));
+		});
```

Comment:
Not a good idea. It's better to have a structure like {"question": [...answers] }
Check migrations for quiz questions